bin/connector_config/Generate.hs

```diff
@@ -37,38 +37,66 @@ data HolyWeapon
 holyWeaponLong :: HolyWeapon -> String
 holyWeaponLong = (++ " Received") . show @HolyWeapon
 
+data FillerItem
+    = AngelicRobe
+    | EnergyRing
+    | SecretBook
+    | Speedwings
+    | GoddessIcon
+    | DragonShield
+    | Talisman
+    | BodyRing
+    | Boots
+    | KnightCrest
+    | HeroCrest
+    | OrionsBolt
+    | GuidingRing
+    | ElysianWhip
+    | OceanSeal
+    | MasterSeal
+    deriving (Show, Enum, Bounded, Typeable)
+
 data Chapter
     = Prologue
     | -- For now, we treat Eirika and Ephraim versions of the same map as the
       -- same location
       C Int
+    | T Int
+    | R Int
     | C5x
     | Endgame
     | Victory
+    | R10
```

Comment:
We don't really need Ruins 10 to be its own location. 5x is just split out because "5x" isn't a number.